### PR TITLE
:ES6: commercial-hosted-youtube ✨

### PR DESCRIPTION
**Pull request by katebee**
## What does this change?

ES6! ✨

- Switch out  'lodash/collections/contains' and 'lodash/collections/forEach' for `includes ` and `Object.keys` respectively
- Remove reliance on `$` from hosted-youtube

## What is the value of this and can you measure success?

ES6! ✨

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/hosted/youtube.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+): void => {
+    const quartile = playerTotalTime / 4;
+    const playbackEvents = {
+        '25': quartile,
+        '50': quartile * 2,
+        '75': quartile * 3,
+    };
+
+    Object.keys(playbackEvents).forEach(key => {
+        const value = playbackEvents[key];
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/commercial/modules/hosted/youtube.js]: You may want to use [`Object.entries`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) for extra awesomeness 😅

File: static/src/javascripts/projects/commercial/modules/hosted/youtube.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                        if (!isDesktop()) {
+                            nextVideoAutoplay.triggerEndSlate();
+                        }
+                    }
+                } else {
+                    // update current time
+                    const currentTime = Math.floor(player.getCurrentTime());
+                    const seconds = currentTime % 60;
+                    const minutes = (currentTime - seconds) / 60;
+                    youtubeTimer.textContent =
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/commercial/modules/hosted/youtube.js]: `${minutes}:${seconds.padStart(2, '0')}` 🌞

Comment (regiskuckaertz) [inline static/src/javascripts/projects/commercial/modules/hosted/youtube.js]: dammit Github 😭

**Comment by regiskuckaertz** [comment]
Fantastic 👍

File: static/src/javascripts/projects/commercial/modules/hosted/youtube.js (hunk 2 above)
Comment (katebee) [inline static/src/javascripts/projects/commercial/modules/hosted/youtube.js]: Would love to convert this to a template literal... but can we use `padStart` if it is currently unsupported in IE?

Comment (regiskuckaertz) [inline static/src/javascripts/projects/commercial/modules/hosted/youtube.js]: PolyfillIO 3.18 [has been released](https://github.com/Financial-Times/polyfill-service/releases/tag/v3.18.0) last week so it should be available now, no?

File: static/src/javascripts/projects/commercial/modules/hosted/youtube.js (hunk 1 above)
Comment (katebee) [inline static/src/javascripts/projects/commercial/modules/hosted/youtube.js]: Niiiice

```
    Object.entries(playbackEvents).forEach(([key, value]) => {
        if (
            !EVENTSFIRED.includes(key) &&
            youtubePlayer.getCurrentTime() > Number(value)
        ) {
            tracking.track(key, atomId);
            EVENTSFIRED.push(key);
            mediator.emit(key);
        }
    });
```
_________

Just tried this out in Safari; it throws: `TypeError: Object.entries is not a function.` 
... so looks like we don't yet have a polyfill for `Object.entries`. 
😞 😞

File: static/src/javascripts/projects/commercial/modules/hosted/youtube.js (hunk 2 above)
Comment (katebee) [inline static/src/javascripts/projects/commercial/modules/hosted/youtube.js]: Just discovered that boot.js loads some es7 polyfills not provided by pollyfill.io... including:

`import 'core-js/modules/es7.string.pad-start';`  🎉💃

**Pull request merged**